src/main/kotlin/uk/gov/justice/digital/hmpps/managerecallsapi/controller/RecallsController.kt

```diff
@@ -92,6 +96,25 @@ class RecallsController(
       )
     )
   }
+
+  @PostMapping("/recalls/{recallId}/assignee")
+  fun assignRecall(
+    @PathVariable("recallId") recallId: RecallId,
+    @RequestBody assignRecallRequest: AssignRecallRequest
+  ): RecallResponse =
+    recallRepository.getByRecallId(recallId)
+      .copy(assignee = assignRecallRequest.assignee.value)
+      .let { recallRepository.save(it) }
+      .toResponse()
+
+  @DeleteMapping("/recalls/{recallId}/assignee")
```

Comment:
Think we agreed (just this morning) to require `{assigneeId}` as ending path param.